src/plugins/event-plugins/JsErrorPlugin.ts

```diff
@@ -6,14 +6,17 @@ export const JS_ERROR_EVENT_PLUGIN_ID = 'com.amazonaws.rum.js-error';
 
 export type PartialJsErrorPluginConfig = {
     stackTraceLength?: number;
+    shouldIgnoreError?: (error: ErrorEvent) => boolean;
 };
 
 export type JsErrorPluginConfig = {
     stackTraceLength: number;
+    shouldIgnoreError?: (error: ErrorEvent) => boolean;
```

Comment:
```suggestion
    filter: (error: ErrorEvent | PromiseRejectionEvent) => boolean;
```

Reply:
From the MDN docs on the `filter` function's callback:
> `callbackFn`
> Function is a predicate, to test each element of the array. Return a value that coerces to true to keep the element, or to false otherwise.
In our case, the `filter` function will be the callback function, but I think the intent will be clear.